### PR TITLE
docs: Deprecate autoinstall guide

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
This feature is unavailable as of #429 

I don't remove the page so as to avoid breaking random links on the internet.

### CODE REVIEW

**Review by CarlosNihelton — APPROVED** [review]
If you rebase over main after merging #423 CI will pass.

**Comment by EduardGomezEscandell** [comment]
Will wait until #423 is merged, there is a silent merge conflict in our custom_wordlist

**Pull request merged**
